web/routes.js

```diff
@@ -2,14 +2,30 @@ import React from 'react';
 import { Route, Switch } from 'react-router';
 
 import LandingPage from 'components/LandingPage';
+import Flex from './components/Flex';
+
+
+/*
+
+-- Routing guide --
+/ - LandingPage
+/team - TeamPage
+/team/:teamMemberId - TeamMemberPage
+/schedule - SchedulePage, with today's day selected by default
+/schedule/:dayOfWeek - SchedulePage, with correct day selected. dayOfWeek will be an integer from 0 to 6 (0 = Monday, 6 = Sunday)
+/schedule/edit/:setblockId - SetblockDetailPage
+
+*/
 
 export default (
-    <div className='routes-container'>
+    <Flex className='routes-container' width='100vw'>
         <Switch>
-            <Route exact path='/' component={LandingPage}/>
-            <Route path='/dashboard' component={LandingPage}/>
-            <Route path='/teamlist' component={LandingPage}/>
+            <Route exact path='/' component={LandingPage} />
```

Comment:
To keep our focus on #4 let's try to nail down the routing foundation and merge this PR.

- Refactor `LandingPage` to `TeamPage`
- Point all routes to their corresponding component (ex: `/schedule` -> SchedulePage) and test that the navigation bar works properly and test all routes render the correct component (it's fine that they are empty still, that will be the next issue)